src/Microsoft.AspNet.Identity/IdentityServiceCollectionExtensions.cs

```diff
@@ -83,18 +83,19 @@ public static IServiceCollection ConfigureIdentityApplicationCookie(this IServic
             services.AddAuthentication();
 
             // Identity services
-            services.TryAdd(ServiceDescriptor.Scoped<IUserValidator<TUser>, UserValidator<TUser>>());
-            services.TryAdd(ServiceDescriptor.Scoped<IPasswordValidator<TUser>, PasswordValidator<TUser>>());
-            services.TryAdd(ServiceDescriptor.Scoped<IPasswordHasher<TUser>, PasswordHasher<TUser>>());
-            services.TryAdd(ServiceDescriptor.Scoped<ILookupNormalizer, UpperInvariantLookupNormalizer>());
-            services.TryAdd(ServiceDescriptor.Scoped<IRoleValidator<TRole>, RoleValidator<TRole>>());
+            services.TryAddSingleton<IdentityMarkerService>();
```

Comment:
I copied what Mvc does, they stick it as a singleton so I just mimiced it:

https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNet.Mvc.Core/DependencyInjection/MvcCoreServiceCollectionExtensions.cs#L124